data-archive/archive-jobs/mosip_credential/mosip_archive_credential_table1.py

```diff
@@ -0,0 +1,139 @@
+#-- -------------------------------------------------------------------------------------------------
+#-- Job Name        : Credential DB Tables Archive
```

Comment:
Also please copy the old mosip_archive_credential_table1.py from 1.1.5.3 also, ad rename it as mosip_archive_credential_table_base-on-no-of-days.py

Reply:
done.